src/components/Page.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+
+const books = [
+  {
+    category: 'Commedy',
+    title: 'Hangover',
+    author: 'Suzanne Collins',
+    progress: {
+      currentChapter: 'Chapter 17',
+      completed: '64',
+    },
+  },
+  {
+    category: 'Olympics',
+    title: 'Tokyo',
+    author: 'Frank Herbert',
+    progress: {
+      currentChapter: 'Chapter 3: "A Lesson Learned"',
+      completed: '8',
+    },
+  },
+  {
+    category: 'Weather',
+    title: 'Capital in the Twenty-First Century',
+    author: 'Suzanne Collins',
+    progress: {
+      currentChapter: 'Introduction',
+      completed: '0',
+    },
+  },
+];
+
+const Page = () => (
+  <div className="books-container">
+    {books.map((element) => (
+      <div className="bookcard" key="element">
+        <div className="book-left">
+          <h4 className="book_genre">{element.category}</h4>
+          <h3 className="title">{element.title}</h3>
+          <h6 className="author">{element.author}</h6>
```

Comment:
- [ ] Instead of directly showing the category, title, and author. send these book data to a different component and then takes the book data as a prop and renders the title, category, and author.

Reply:
### Hello @TSHEPO-CLOUD.

- I do agree with my  TSE colleague, instead, you can move these constants to book.js then call it and add props to make it more convenient. [see this article](https://reactjs.org/docs/components-and-props.html)

![kkkkk](https://user-images.githubusercontent.com/61562626/139855857-5e9b9948-2393-4d35-b3bf-de07f1a0e5d2.jpeg)